src/sensors/LaserRangeFinder.cpp

```diff
@@ -64,26 +64,26 @@ void LaserRangeFinder::RenderResult::renderLine(const Vec2& p1, const Vec2& p2)
     // In the latter case, we may need to render two parts
 
     uint i_min1, i_max1, i_min2, i_max2;
```

Comment:
would it not be easier to make these regular int rather than do all the casting below?

Reply:
These are eventually used to iterate. So you could only reduce the number of casts from 4 to 2. But I think it is better to keep these uint, So it is very clear these shouldn't be negative and therefore not lie outside the sensor range.